### PR TITLE
[zh-cn]Reference files to sync task 9

### PR DESCRIPTION
**Pull request by asa3311**
Reference files to sync task 9

Related to task-9 of issue https://github.com/kubernetes/website/issues/39987.

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-join_etcd.md [hunk 1]
```diff
@@ -85,6 +85,20 @@ Create a new control plane instance on this node
...
+<tr>
+<td colspan="2">--dry-run</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+Don't apply any changes; just output what would be done.
+-->
+<p>
+不要做任何更改；只输出将要做的事情。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-join_etcd.md:97]:
```suggestion
不做任何更改；只输出将要执行的操作。
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_kubelet-start.md [hunk 2]
```diff
@@ -126,6 +126,20 @@ For token-based discovery, allow joining without --discovery-token-ca-cert-hash
...
+<tr>
+<td colspan="2">--dry-run</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+Don't apply any changes; just output what would be done.
+-->
+<p>
+不要做任何更改；只输出将要做的事情。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_kubelet-start.md:138]:
```suggestion
不做任何更改；只输出将要执行的操作。
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**